drizzlepac/haputils/photometry_tools.py

```diff
@@ -150,7 +150,9 @@ def iraf_style_photometry(phot_apertures, bg_apertures, data, photflam, photplam
         if math.isclose(photflam, 0.0, abs_tol=constants.TOLERANCE):
             mag_err = mag
         else:
-            mag_err = 1.0857 * flux_error / flux
+            # ignores warnings from log10 of zeros and negative values
+            with np.errstate(divide='ignore', invalid='ignore'):
+                mag_err = 1.0857 * flux_error / flux
```

Comment:
It seems like the flux here is the troublesome variable as there is no log computation.  What is the value of mag_err here after the divide by zero?